### DISCUSSION TITLE
Will a non-matching alt allele cause a position to fail to match on the reference panel?

### DISCUSSION THREAD

**Discussion by Fiwx** [PGScatalog/pgsc_calc#253] [Q&A] [answered]
Sometimes, the VCFs being ran have a homozygous reference allele at a position with an alternate allele which doesn't match the alternate allele at that position in the reference panel (e.g., 1000 Genomes).

Will this cause matching to fail? (E.g., for ancestry predictions / adjustments.)

Example:
VCF: 0/0, reference: A, alternate: T. (Genotype: AA.)
Reference panel: mostly 0/0, reference: A, alternate: G.

**Comment by smlmbrt** [comment, marked as answer]
Yes, this example would not be identified as an intersecting variant between the two panels. The intersection also doesn't consider strand flips (as the reference panels are build-specific and already reconciled).
